data/json/monster_special_attacks/spells.json

```diff
@@ -485,5 +485,49 @@
     "type": "ter_furn_transform",
     "id": "portal_cage_revert_transform",
     "furniture": [ { "result": [ "f_null" ], "valid_furniture": [ "f_absence" ] } ]
+  },
+  {
+    "id": "boomer_claymore_initial",
+    "type": "SPELL",
+    "//": "exist because zed try to use boomer_claymore on max range except trying to minimize the distance (ty venera for idea)",
+    "name": { "str": "Initial boomer claymore attack" },
+    "description": "Nothing to see here.",
+    "valid_targets": [ "hostile" ],
+    "effect": "attack",
+    "shape": "blast",
+    "min_damage": 0,
+    "max_damage": 0,
+    "min_range": 3,
+    "max_range": 3,
+    "extra_effects": [ { "id": "boomer_claymore" } ]
+  },
+  {
+    "id": "boomer_claymore",
+    "type": "SPELL",
+    "name": { "str": "Boomer claymore attack" },
+    "description": "Nothing to see here.",
+    "valid_targets": [ "hostile", "ally", "ground" ],
+    "effect": "attack",
+    "shape": "cone",
+    "min_damage": 30,
+    "max_damage": 30,
+    "damage_type": "biological",
```

Comment:
Regular explosions deal bash damage in the code.

And it would probably be smart to make the explosion deal a random amount of damage between 10-30 damage instead of just flat damage amount. You should probably consider if you want this effect to instakill fragile characters, IRC it will do so with current stats.

Reply:
Already replase it with cut damage

Reply:
Also remember its pretty rare monster, and has not so much damage against armored monsters

Reply:
Should probably replace it with bash just for consistency sake.

Reply:
Explosion has "bullet" damage, which not exist in the spell system
Also:
![IMG_20220121_061306](https://user-images.githubusercontent.com/67688115/150469880-f6c1a8eb-5f2d-4ae7-84bd-fa18f02e91cb.jpg)

Reply:
Well the explosion "blast" which always hits you and works mostly like the spell here deals bash. Shrapnel does deal bullet damage, but is also not guaranteed to always hit you.

You can actually make spells  cast normal explosions using the "explosion" spell effect. Although Im not sure if you considered this alternative already

Reply:
I never check how it works with shapes except "blast"
Ill test it and change the effect if thats would be better than current solution

Reply:
Oh btw, if you misunderstood it (as pryanik on the comment below) there is no real landmineon his ribcage, its just a dummy name

Reply:
just made some test, and of course `explosion` effect has no support for `shape`